src/packages/client/src/utils/generateInFolder.ts

```diff
@@ -74,10 +74,15 @@ export async function generateInFolder({
 
   let runtimePath
   if (useLocalRuntime) {
-    runtimePath = path.relative(outputDir, path.join(__dirname, '../runtime'))
-  }
-  if (useBuiltRuntime) {
+    if (useBuiltRuntime) {
     runtimePath = path.relative(outputDir, path.join(__dirname, '../../runtime'))
+
+    } else {
+    runtimePath = path.relative(outputDir, path.join(__dirname, '../runtime'))
+
```

Comment:
no prettier here? 😄

Reply:
Haha good point. Disabled it. Will enable it again.